include/ignition/gui/qml/IgnCard.qml

```diff
@@ -74,6 +74,11 @@ Pane {
    */
   property string closeIcon: "\u2715"
 
+  /**
+   * The plugin backgroung color
+   */
+  property string cardBackground: "#ffffff"
```

Comment:
Can we keep the default value as `transparent`? Otherwise with this change `WorldStats` and `WorldControl` look like this:

![whitebg](https://user-images.githubusercontent.com/5751272/81761460-d6dfc200-947e-11ea-9548-3589a891ac32.png)

Reply:
Okay. I'll do that.